### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Shubham-Porwall**
Hi Kalpana Ma'am,

Creating this pull request for reviewing the solutiobn

### CODE REVIEW

File: file_manage.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+afile = File.open("test.rb","r+")
```
Comment (kalpana19sharma) [inline file_manage.rb]: Space after `,`

File: file_manage.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+if afile
+    afile.syswrite('hello GAMMASTACK')
+else
```
Comment (kalpana19sharma) [inline file_manage.rb]: Use 2 space indentation and fix this in other files as well.

File: file_manage.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+bfile = File.open("test.rb","r")
+if bfile
+    con = bfile.sysread(50)
+    puts con
+else
+    puts "Unavailable"
+end
+a = 100
+puts "Hello #{a}"
```
Comment (kalpana19sharma) [inline file_manage.rb]: Add a new line at the end of the file. And handle this in all files.

File: join_array.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+a = Array[1,2,3,4]
+b = Array[5,6,7,8]
+c = a+b;
+puts c * " "
```
Comment (kalpana19sharma) [inline join_array.rb]: Use proper spaces

**Review by kalpana19sharma — CHANGES_REQUESTED** [review]
@Shubham-Porwall Use proper heading for your PR.

**Comment by Shubham-Porwall** [comment]
@kalpana19sharma Changes is been performed.

File: join_array.rb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+a = Array[1,2,3,4]
+b = Array[5,6,7,8]
```
Comment (kalpana19sharma) [inline join_array.rb]: Use spaces after `,`

Comment (kalpana19sharma) [reply join_array.rb]: You can eliminate `Array` class here
```suggestion
a = [1, 2, 3, 4]
b = [5, 6, 7, 8]
```

Comment (kalpana19sharma) [reply join_array.rb]: And use proper variable names.

File: map_method.rb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+def xyz (g)
+  return g.to_i * 2
+end
+a = Array[1,2,3,"5"]
+b = a.map{|i|  xyz(i)}
```
Comment (kalpana19sharma) [inline map_method.rb]: 1. Use proper method and variable names
2. Use proper spaces around operators

File: no_of_word.rb [hunk 7]
```diff
@@ -0,0 +1,10 @@
+str=gets.chomp
+user = Hash.new
+for i in(0...str.length)
+    if user[str[i]].nil?
+        user[str[i]] = 1
+    else
+        user[str[i]] += 1
+    end
+end
```
Comment (kalpana19sharma) [inline no_of_word.rb]: 1. Use 2 space indentation
2. Use proper variable names
3. Is there any in-built method in ruby to find the frequency

File: series_printing.rb [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+class Test
+  def p(a,b)
+    for i in a..b
+      print (i.to_s+" ")
+    end
+  end
+end
+
+user = Test.new
+user.p(1,100)
```
Comment (kalpana19sharma) [inline series_printing.rb]: Use proper spaces around operators.

**Review by kalpana19sharma — CHANGES_REQUESTED** [review]

File: file_manage.rb [hunk 9]
```diff
@@ -0,0 +1,15 @@
+afile = File.open("test.rb" , "r+")
```
Comment (kalpana19sharma) [inline file_manage.rb]: No need to add space before `,`. Please read the guide carefully. ```Use spaces around operators, after commas, colons and semicolons, around { and before }.```

File: file_manage.rb [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+if afile
+  afile.syswrite('hello GAMMASTACK')
```
Comment (kalpana19sharma) [inline file_manage.rb]: Either use `' '` or use `" "`. Don't use both untill and unless its necessary. We generally prefer `' '`.

**Review by kalpana19sharma — CHANGES_REQUESTED** [review]

**Review by kalpana19sharma — APPROVED** [review]

**Pull request merged**